tests/test_main.py

```diff
@@ -147,6 +181,76 @@ def test_M852(self):
         g.write('M500')
         g.move(Z=5)
 
+    # REQUIRES PRINTER CONNECTION
+    def test_M236(self):
+        g = self.g
+
+        # Pressure Definitions
+        pressure_setpoint = 10
+        pressure_house_air = 45
+        pressure_buffer = 2
+        pressure_upper_bound = 131
+        pressure_lower_bound = -1
+        pressure_near_zero = 2
+
+        # Assert that available output pressure is 0 psi when tank near 0 psi
+        status = g.write('M236 V', resp_needed=True)
+
+        # Check that tank is near 0 psi first
+        if (status.split()[7] <= pressure_near_zero):
```

Comment:
This is checking that a string is less than or equal to an integer, which will always be false. The warning will always be printed.